master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -138,7 +138,7 @@ def test_start_vpc_instance(self):
         bs = ec2.EC2LatentWorker('bot1', 'sekrit', 'm1.large',
                                  identifier='publickey',
                                  secret_identifier='privatekey',
-                                 keypair_name="test_key",
+                                 keypair_name="latent_buildbot_slave",
```

Comment:
Default can't be changed due to backward compatibility.
Current use of default value is deprecated and warning is emitted, if user doesn't explicitly specify value for `keypair_name` or `security_name`.

Please change value that is used in test.